medium-tests/src/test/java/mediumtest/fixtures/ServerFixture.java

```diff
@@ -562,6 +572,12 @@ private void registerIssueAnticipateTransitionResponses() {
     }
 
     private void registerSourceApiResponses() {
+      sourceFileByComponentKey
```

Comment:
again, not ideal but we don't even have the issue here so doing it the proper way would require more work. Happy to accept suggestions/help

Reply:
Yep, in fact the `withSourceFile` should be located in the branch builder and not in the server builder. Up to you, depends if you feel brave :)